napari/layers/utils/stack_utils.py

```diff
@@ -0,0 +1,188 @@
+from napari.utils.misc import ensure_iterable, ensure_sequence_of_iterables
+
+
+def split_channels(
+    data: np.ndarray, channel_axis: int, **kwargs,
+) -> List[Tuple]:
+    """Function to split the data array into seperate arrays along an axis
+
+    Parameters
+    ----------
+    data : array or list of array
+    channel_axis : int
+        Axis to split the image along.
+    kwargs: keyword arguments
+
+    Returns
+    -------
+    List of LayerData tuples: [(data: array, metadata: Dict, type: str )]
+
+    """
+
+    # Determine if data is a multiscale
+    multiscale = kwargs['multiscale']
+    if multiscale is None:
+        multiscale, data = guess_multiscale(data)
+    kwargs['multiscale'] = multiscale
+    n_channels = (data[0] if multiscale else data).shape[channel_axis]
+    kwargs['blending'] = kwargs['blending'] or 'additive'
+
+    # these arguments are *already* iterables in the single-channel case.
+    iterable_kwargs = {'scale', 'translate', 'contrast_limits', 'metadata'}
+
+    # turn the kwargs dict into a mapping of {key: iterator}
+    # so that we can use {k: next(v) for k, v in kwargs.items()} below
+    for key, val in kwargs.items():
+        if key == 'colormap' and val is None:
+            if n_channels == 1:
+                kwargs[key] = iter(['gray'])
+            elif n_channels == 2:
+                kwargs[key] = iter(colormaps.MAGENTA_GREEN)
+            else:
+                kwargs[key] = itertools.cycle(colormaps.CYMRGB)
+
+        # make sure that iterable_kwargs are a *sequence* of iterables
+        # for the multichannel case.  For example: if scale == (1, 2) &
+        # n_channels = 3, then scale should == [(1, 2), (1, 2), (1, 2)]
+        elif key in iterable_kwargs:
+            kwargs[key] = iter(ensure_sequence_of_iterables(val, n_channels))
+        else:
+            kwargs[key] = iter(ensure_iterable(val))
+
+    layerdata_list = list()
+    for i in range(n_channels):
+        if multiscale:
+            image = [
+                np.take(data[j], i, axis=channel_axis)
+                for j in range(len(data))
+            ]
+        else:
+            image = np.take(data, i, axis=channel_axis)
+        i_kwargs = {}
+        for key, val in kwargs.items():
+            try:
+                i_kwargs[key] = next(val)
+            except StopIteration:
+                raise IndexError(
+                    "Error adding multichannel image with data shape "
+                    f"{data.shape!r}.\nRequested channel_axis "
+                    f"({channel_axis}) had length {n_channels}, but "
+                    f"the '{key}' argument only provided {i} values. "
+                )
+
+        layerdata = (image, i_kwargs, 'image')
+        layerdata_list.append(layerdata)
+
+    return layerdata_list
+
+
+def stack_to_images(stack: Image, axis: int, **kwargs: Dict,) -> List[Image]:
+    """Function to split the active layer into separate layers along an axis
```

Comment:
You can leave out the "Function to" part from all the docstrings

Reply:
also, this function has no concept of an "active" or "inactive" layer.  So the docstring should be more specific.  "Splits a single `Image` layer into a list of Image layers along `axis`"